Source/KerbalCombatSystems/Debug.cs

```diff
@@ -12,13 +12,18 @@ public class KCSDebug : MonoBehaviour
         public static bool showLines;
         private static List<LineRenderer> lines;
         private static List<float> times;
+
+        //relavent game settings
+        private int refreshRate;
```

Comment:
I don't think there's a benefit to using refresh rate in this file. I'd rather it repeat at a known rate over something set by the user, considering it doesn't affect performance.

Reply:
I believe this was an inherited block and most likely not relavant after end user linedraws got moved elsewhere so this whole performance draw is toggleable